src/app/error-display/error-display.component.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+angular.module('esn.calendar.event-consultation')
+  .component('calEventExternalErrorDisplay', {
+    bindings: {
+      error: '<',
+    },
+    controllerAs: 'ctrl',
```

Comment:
where is controller?

Reply:
This component doesn't need controller. It just displays the error that is received from the parent component

Reply:
clean it then

Reply:
I mean if there is no controller, so no need controllerAs ;)

Reply:
Ah. Ok then!